makefile

```diff
@@ -0,0 +1,19 @@
+dkc := "metabase/metabase.yml"
+
+up:
+	docker-compose -f ${dkc} up --build
+
+network:
+	docker network create metabase
+
+clean:
+	docker-compose -f ${dkc} kill
+	docker-compose -f ${dkc} stop
+	docker-compose -f ${dkc} down --rmi local --volumes
+	docker-compose -f ${dkc} rm -f
+	@echo "Docker containers have been stopped and deleted."
+
+migrate:
+	@echo "Loading data into the database..."
+	docker cp data/data_example.csv mongo:/
```

Comment:
Probably we will need to add explanation about this line! What do you think to create a `make command` where the user can pass the file path?

Reply:
added parameter in make migrate command